### ISSUE TITLE
Do Lotek SRX recievers need to be configured to decode newer codesets?

### ISSUE THREAD

**Issue by leberrigan** [MotusDev/Motus-TO-DO#806]
The typical DTA file provided by a SRX-series receiver will begin with the following header: 

> SRX600 Information:
> Environment History:
> changed: 11/15/18  19:32:37
> 
> Master Firmware: Version 8.3
> Slave Firmware:  Version 12.0
> Receiver S/N:    6455
> 
> Active Configuration: 1
> Site Code: 1
> Code Set: Lotek4

The last part appears to state the code set it's listening to. What I'm wondering is:
- Do all these receivers require updated firmware to listen for the '**Lotek6m**' code set? I.e., will newer tags be missed if the firmware is not updated?
- Will this code set be available for all SRX-series receivers, or only the 800 series?
- I was just sent file with 'Code Set: Lotek3' in the header - should we be concerned that this receiver won't pick up anything from 'Lotek4'?

**Comment by denislepage** [comment]
Yes, a firmware upgrade will be needed. Receivers without the patch will be able to pick up the first 512 numbers that overlap the Lotek4 codeset, but not the additional ~200 new values.

Even if the tags are registered as Lotek6M in Motus, and the SRX is looking for Lotek4 and reporting that in the data file, I am pretty sure tag finder will revert to looking at Lotek6M as an alternative. I don’t think it will break if it sees the tag being Lotek4 where the database says it’s actually Lotek6M. We should ensure of that.

Sean/Dave didn’t confirm whether the 600 series will support the new code set. I think they said “yes, but it will need to be tested”, and that that limited memory on the units would mean no longer looking for some tags that aren’t used by Motus, which would be fine. We’ll need to follow up on that.

I don’t know about Lotek3. We don’t have any Lotek3 tags and haven’t in a while, that I know of. If the receiver is only looking for Lotek3, it is likely not finding anything. Is this a setting that users control? A receiver that hadn’t been updated in years, or?
